### DISCUSSION TITLE
Email plugin improvements

### DISCUSSION THREAD

**Discussion by gempain** [medusajs/medusa#6921] [Feature Requests]
Hi 👋 

The (sendgrid plugin)[https://docs.medusajs.com/plugins/notifications/sendgrid] is not very practical.

It'd be nice to:
- make it generic for other providers than sendgrid (just configure a nodemailer transport)
- provide default templates (in English) for most events, this would save dev time, as we probably write relatively the same thing

Keep up the great work !

**Comment by namnguyenn17** [comment]
same issue

**Comment by kriss145** [comment]
@gempain I'm working with the plugin where we will have ready all templates out of the box and also more another features which are on my roadmap. If u want some features, add your ideas in the discussions page :)

https://github.com/codee-sh/medusa-plugin-notifications
